multiverse:
  services:
    query_data:
      port: 7400

  publishers:
    tf:
      port: 7300
      rate: 60
    odom:
      port: 7301
      rate: 60
      body: tiago_dual

  subscribers:
    cmd_vel:
      port: 7200
      body: tiago_dual